playing_game.py

```diff
@@ -0,0 +1,43 @@
+#!/bin/python3
+
+import os
+import sys
+
+#
+# Complete the gamingArray function below.
+#
+def gamingArray(arr):
+    #
+    # Write your code here.
+    #
+    moves = 0
+    max_arr = 0
+    n = len(arr)
+    # go through the list
+    for i in range(n):

```

Comment:
This is wrong. The array size can be reduced when removing the elements to the right of the maximum.